stylesheets/footer.css

```diff
@@ -24,6 +24,12 @@ p {
   font-family: 'Roboto', sans-serif;
 }
 
+#error {
+  color: red;
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+}
+
```

Comment:
Nice work done ✔Project is completed, you can merge this branch, and your project is **### approved** 😁👍

Congratulations !! 👍 🎉✔